misocho/questioner
# #163149110 Refactors code

#### What does this PR do?
Refactors code and implements oop
#### Description of Task to be completed?
* Refactor code
#### How should this be manually tested?
* Clone repo https://github.com/misocho/questioner.git
* Create a virtual environment
* Activate the 
* Branch to ch-api-oop-163149100
* Pull from ch-api-oop-163149100virtual enviroment
* Pip install requirements.txt
* Check oop
* run pytest for testing
#### What are the relevant pivotal tracker stories?
#163149110
https://www.pivotaltracker.com/n/projects/2235306
